dotcom-rendering/src/web/components/Island.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+
+import { Placeholder } from './Placeholder';
+
+type When = 'idle' | 'visible';
+
+interface HydrateProps {
+	deferUntil?: When;
+	clientOnly?: false;
+	placeholderHeight?: undefined;
+	children: JSX.Element;
+}
+
+interface ClientOnlyProps {
+	deferUntil?: When;
+	clientOnly: true;
+	placeholderHeight: number;
```

Comment:
In the case of something like a fixed position banner (e.g. `StickyBottomBanner`) would the right thing to do be pass 0 for the placeholder height? Or are there perhaps cases (like this) where a placeholder height doesn't make sense? (Does this maybe imply it should be optional here too?)

Reply:
🤔 that's a good point. You should be able to specify `clientOnly` as true but just not have a placeholder. Needing to set it as zero is cumbersome.

Reply:
I've marked this prop as optional 👍